crates/l2/Cargo.toml

```
@@ -6,9 +6,14 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+reqwest = { version = "0.12.7", features = ["json"] }
 tokio.workspace = true
 tokio-util.workspace = true
 tracing.workspace = true
+serde = { version = "1.0.203", features = ["derive"] }
+serde_json = "1.0.117"
```

These are also in the workspace
```suggestion
serde.workspace = true
serde_json.workspace = true
```